Finjector.Web/Controllers/UserController.cs

```diff
@@ -41,7 +41,13 @@ public IActionResult Info()
             return Challenge(); // trigger authentication, but claims should never be null here
         }
 
-        return Ok(claims.ToDictionary(c => c.Type, c => c.Value));
+        var dict = claims.ToDictionary(c => c.Type, c => c.Value);
+        
+        var iamId = dict.Where(a => a.Key == "ucdPersonIAMID").Select(a => a.Value).FirstOrDefault();
```

Comment:
use `IamIdClaimFallbackTransformer.ClaimType` instead

Reply:
Also I'd do it like this: `var iamId = dict.First(a => a.Key == IamIdClaimFallbackTransformer.ClaimType).Select(a => a.Value);`

We don't have to worry about it being missing, if so there is a much bigger problem!